kyaml/fn/framework/example_mutate_comments_test.go

```diff
@@ -0,0 +1,36 @@
+package framework_test
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"sigs.k8s.io/kustomize/kyaml/fn/framework"
+	"sigs.k8s.io/kustomize/kyaml/fn/framework/command"
+	"sigs.k8s.io/kustomize/kyaml/yaml"
+)
+
+// In this example, we mutate line comments for field metadata.name.
```

Comment:
It's difficult to understand the use-case for it ? Can we lead the description in what cases such a functionality is required ?